### PR TITLE
Misc pyverbs tests fixes round 2

### PR DESCRIPTION
**Pull request by galpress**
Another set of pyverbs fixes which allow tests of unsupported devices to skip.

### CODE REVIEW

File: tests/test_device.py [hunk 1]
```diff
@@ -71,15 +71,16 @@ def test_query_gid(self):
                 ctx.query_gid(port_num=1, index=0)
 
     @staticmethod
-    def verify_device_attr(attr):
+    def verify_device_attr(attr, device):
         """
         Helper method that verifies correctness of some members of DeviceAttr
         object.
         :param attr: A DeviceAttr object
```
Comment (EdwardSro) [inline tests/test_device.py:78]: please add the device param to the docstring

File: tests/test_device.py [hunk 2]
```diff
@@ -71,15 +71,16 @@ def test_query_gid(self):
...
         :return: None
         """
-        assert attr.node_guid != 0
-        assert attr.sys_image_guid != 0
+        if device.node_type != 'Unknown':
```
Comment (EdwardSro) [inline tests/test_device.py:81]: do you mean "IBV_NODE_UNKNOWN" ?
node_type is not a string (it's just printed as a string using an internal translation function)

File: tests/test_mr.py [hunk 3]
```diff
@@ -219,7 +219,9 @@ def test_reg_mw_wrong_type(self):
                 try:
                     mw_type = random.randint(3, 100)
                     MW(pd, mw_type)
-                except PyverbsRDMAError:
+                except PyverbsRDMAError as ex:
+                    if ex.error_code == errno.EOPNOTSUPP:
+                        raise unittest.SkipTest('Create memory window of type {} is not supported'.format(mw_type))
```
[the remark this answers is not in the document]
Comment (galpress) [reply tests/test_mr.py:224]: Ack

File: tests/test_mr.py [hunk 4]
```diff
@@ -219,7 +219,9 @@ def test_reg_mw_wrong_type(self):
...
                     pass
```
[the remark this answers is not in the document]
Comment (galpress) [reply tests/test_mr.py:225]: Ack

File: tests/test_device.py (hunk 1 above)
Comment (galpress) [reply tests/test_device.py:78]: Ack

File: tests/test_device.py (hunk 2 above)
Comment (galpress) [reply tests/test_device.py:81]: Ack

**Pull request merged**